### DISCUSSION TITLE
Customise how Select options are saved (preserve key/values)

### DISCUSSION THREAD

**Discussion by joshembling** [filamentphp/filament#9494] [General]
If I have a simple select:

```php
Select::make('status')
    ->options([
        'draft' => 'Draft',
        'reviewing' => 'Reviewing',
        'published' => 'Published',
    ])
    ->multiple()
```

When I want to see the data that was saved after the form submission I get something like: 
```php
'status' => [
     0 => 'published',
     1 => 'reviewing'
];
```

Is there a way to somehow save the data so that the key/value pair are maintained so I'd have something like this instead: 
```php
'status' => [
     'published' => 'Published',
     'reviewing' => 'Reviewing'
];
```

**Comment by sinnbeck** [comment]
Can you show your code for saving it (mutator or cast)? Is it a json column in the database ?
